internal/tools/clickhouse_create_transactions_table.sql

```diff
@@ -19,11 +19,5 @@ CREATE TABLE base.transactions (
     `insert_timestamp` DateTime DEFAULT now(),
     INDEX block_timestamp_idx block_timestamp TYPE minmax GRANULARITY 1,
     INDEX transaction_hash_idx hash TYPE bloom_filter GRANULARITY 1,
-) ENGINE = SharedReplacingMergeTree(
-    '/clickhouse/tables/{uuid}/{shard}',
-    '{replica}',
-    insert_timestamp,
-    is_deleted
-)
-ORDER BY (chain_id, block_number) SETTINGS index_granularity = 8192
-SETTINGS allow_experimental_replacing_merge_with_cleanup = 1;
\ No newline at end of file
+) ENGINE = ReplacingMergeTree(insert_timestamp, is_deleted)
+ORDER BY (chain_id, block_number, hash) SETTINGS allow_experimental_replacing_merge_with_cleanup = 1;
```

Comment:
is it possible to order by hash? and why do we need ordering by chainId? each chain has it's own table anyways

Reply:
we have a index for hash.
We have chain_id everywhere in the table schema, that is why. We don't necessarily need separate tables with this schema

Reply:
this means the tables are going to have billions of rows and have to handle inserts from multiples indexer instances in parallel